### DISCUSSION TITLE
DD8216BD-689D-C6C6-87A8-1705B40894B1

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#135] [General]
# DD8216BD-689D-C6C6-87A8-1705B40894B1

This addon adds fictional (but realistic) textures by Snowfall-The-Cat for the dwarf planet Haumea and its rings. It also simulates the precession of Haumea's rings.

https://celestia.mobi/addon?item=DD8216BD-689D-C6C6-87A8-1705B40894B1

**Comment by Lil-Gdops** [comment]
Its broken, an black thing is making it cut in half

**Reply by gtfggjbdrff** [reply]
This plugin has been fixed

**Reply by gtfggjbdrff** [reply]
If this problem persists, it indicates that your Celestia version is low. It is recommended to upgrade to version v1.7.1.
